runtime-web/libwapcaplet/autobuild/defines

```diff
@@ -4,6 +4,8 @@ PKGDEP="glibc"
 BUILDDEP="netsurf-buildsystem"
 PKGDES="A C library for string internment"
 
+ABSPLITDBG=0
```

Comment:
This is wrong - if this package contains any ELF file(s), Autobuild should be able to generate debug symbols. Otherwise, check if Autobuild flags were passed correctly.